ocs_ci/ocs/resources/pvc.py

```diff
@@ -190,18 +210,21 @@ def get_all_pvcs(namespace=None, selector=None):
     Gets all pvc in given namespace
 
     Args:
-        namespace (str): Name of namespace
+        namespace (str): Name of namespace  ('all-namespaces' to get all namespaces)
         selector (str): The label selector to look for
 
     Returns:
          dict: Dict of all pvc in namespaces
     """
+    all_ns = False
+    if namespace == 'all-namespaces':
```

Comment:
Can be compacted - 
`all_ns = True if lower(namespace) == 'all-namespaces' else False`

Reply:
again, readability